### PR TITLE
Xiaoya add fine-tuning support for DINO

### PR DESCRIPTION
**Pull request by xiaoyachong**
1. Added DINOv3-Seg ([DINOv3](https://github.com/facebookresearch/dinov3) + [EOMT](https://github.com/tue-mps/eomt) mask module) fine-tuning for semantic segmentation. We use [LightlyTrain](https://github.com/lightly-ai/lightly-train) framework for finetuning.
image

 Algorithm registration is implemented for three DINOv3 ViT-S models:
   * `dinov3/vits16-eomt`
   * `dinov3/vits16-eomt-coco`
   * `dinov3/vits16-eomt-cityscapes`
     
 These correspond to:
 * DINOv3 + EOMT mask module (not fine-tuned)
 * DINOv3 + EOMT mask module fine-tuned on COCO-Stuff
 * DINOv3 + EOMT mask module fine-tuned on Cityscapes

 The input image sizes for the three models are:
 * 512
 * 518
 * 1024

In our testing, `dinov3/vits16-eomt-cityscapes` generally provides the best results, although it requires longer fine-tuning time. For a dataset with 42 images, approximately 5k steps are needed for convergence, which takes around 3 hours using a single GPU. At least 1k steps are typically required to obtain reasonable results when trained on 1 annotated image (results attached below). If a GPU is not available, users can run 100 steps for simple testing, although the resulting metrics will likely be very low.

image

The default `TUNet` and `dinov3/vits16-eomt` models have approximately 1.93M and 23.29M parameters, respectively. Therefore, ViT-S is much larger and generally requires a larger training dataset, more GPU memory, and longer training time.

The full list of supported models can be found in the [LightlyTrain model documentation](https://docs.lightly.ai/train/stable/semantic_segmentation.html#model). Additional backbones can also be integrated into our application by simply adding them to `models.json`. However, during testing, the ViT-S models provided good performance, especially since most user-annotated datasets are relatively small (typically fewer than 100 images). Using larger backbones with such limited datasets may otherwise lead to lower performance and less stable results.

2. Added a `lightly_train` PNG as model info in UI.

image

3. SAM3 was not added at this stage because instance segmentation is not required for most user use cases. 

Supporting SAM3 would require:
- instance masks,
- semantic masks,
- and conversion to COCO format.

This functionality can be implemented in a future PR if needed.

> Companion PRs:
> mlex_lightlytrain: https://github.com/mlexchange/mlex_lightlytrain_segmentation_prototype/pull/1
> mlex_tomo: https://github.com/mlexchange/mlex_tomo_framework/pull/18
> mlex_utils: https://github.com/mlexchange/mlex_utils/pull/6

### CODE REVIEW

File: callbacks/control_bar.py [hunk 1]
```diff
@@ -1048,3 +1048,25 @@ def validate_dilation_array(dilation_array):
     except ValueError:
         # If there's any error in parsing or validation, return False
         return "Provide a list of ints for dilation"
+
+
+# CHANGED: new callback to update model info image and link based on selected model
+MODEL_INFO_MAP = {
```
Comment (taxe10) [inline callbacks/control_bar.py]: As we increase the number of algorithms, I wonder if this is going to be difficult to maintain. Could we move this logo to the algorithm assets in mlflow?

**Review by taxe10 — COMMENTED** [review]
Looks good! Left a minor comment

File: callbacks/control_bar.py (hunk 1 above)
Comment (xiaoyachong) [reply callbacks/control_bar.py]: Thanks for your suggestion! I also log the logo to `MLflow` in the latest version, and update mlex_utils: https://github.com/mlexchange/mlex_utils/pull/6.

**Review by taxe10 — APPROVED** [review]
